File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopDetailParam.java
package com.cakk.domain.mysql.dto.param.shop;

import java.util.Set;

import com.cakk.common.enums.Days;

public record CakeShopDetailParam(
	Long cakeShopId,
	String shopName,
	String thumbnailUrl,
	String shopBio,
	String shopDescription,
	Set<Days> operationDays,
	Set<CakeShopLinkParam> links
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopDetailParam.kt
package com.cakk.domain.mysql.dto.param.shop

import com.cakk.common.enums.Days

data class CakeShopDetailParam(
	val cakeShopId: Long,
	val shopName: String,
	val thumbnailUrl: String,
	val shopBio: String,
	val shopDescription: String,
	val operationDays: Set<Days>,
	val links: Set<CakeShopLinkParam>
)
